src/classifier/Pipe.cpp

```diff
@@ -224,6 +231,11 @@ void Pipe::TrainEpoch(int epoch) {
   int time_scores = 0;
   int num_mistakes = 0;
 
+  if ( epoch == 0 ){
+    LOG(INFO) << "Lambda: " << lambda << "\t"
+    << "Regularization Constant: " << options_->GetRegularizationConstant() << "\t"
+    << "Num instances: " << num_instances << endl;
+  }
```

Comment:
Regularization Constant -> "Regularization constant"
Num instances -> "Number of instances" (spelled out)